### DISCUSSION TITLE
Excesso de empresas de recrutamento e seleção (?)

### DISCUSSION THREAD

**Discussion by willycamargo** [frontendbr/forum#556] [Discussão]
Faz um bom tempo que venho percebendo uma certa "epidemia de recrutamento" na área de TI.

Existem inúmeros sistemas de contratação e constantemente vemos novos sendo lançados, inclusive em um ou outro bate papo por ai acabo vendo o comentário de algum founder de uma nova startup/empresa de recrutamento.

Como exemplos, posso destacar:

[Trampos.co](https://trampos.co)
[Facebook Jobs](http://www.facebook.jobs)
[Hipsters ponto JOBS](http://hipsters.jobs)
[Linkedin Jobs](https://www.linkedin.com/jobs)
[Top Tal](https://www.toptal.com)
[BairesDev](https://www.bairesdev.com) - Esses já me enviaram alguns email
[Recrutando](http://www.recrutando.com) - Esse eu nem conhecia, encontrei pesquisando no google
[Jobs Londrina](http://jobslondrina.com) - Esse é conhecido em minha cidade e aposto que existe um parecido para sua cidade

Claro que nessa lista não estão todas existentes e claro que não vou fazer um `awesome-recruitment-systems` (nem sei se já existe). Mas quem quiser postar abaixo outros exemplos, fique a vontade.

Talvez essa minha percepção e incômodo tenha sido aguçado de acordo com que fico mais próximo de comunidades, mas talvez não.

O que quero saber é a visão de vocês quanto a quantidade e qualidade dessas empresas. Seja aquelas que trabalham apenas com  ligação entre o profissional e a empresa ou seja ela estilo Marketplace (como a TopTal).

Quem de vocês já utilizou o serviço dessas empresas/plataformas? O que acharam?

---

Edit 1:
Não considerei nessa issue a existência de grupos nas redes sociais que tratam exatamente da mesma coisa (Recrutamento de pessoas especializadas). Mas é um ponto para debater também.

**Comment by didiraja** [comment]
Pra esse tipo de coisa, realmente seria muito melhor um grande site que gerenciasse todas ou a maioria das vagas. Mas a realidade (e acho que nem tem a ver com a área) é que temos milhões de sites e cada um deles tem um peso diferente entre qualidade, sem contar que essas coisas mudam com o tempo.

Os sites que eu olhava na época de estágio eram uns, e os que vejo recentemente procurando emprego são outros. Inclusive achei esse repositório aqui, justamente perguntando sobre onde procurar vagas de front-end.

Enquanto não houver uma centralização, continua sendo melhor pro candidato conhecer todos os milhões de sites, verificá-los um a um e fazer essa diferenciação do peso de maneira pessoal.

E similar á esse de Londrina, aqui no RJ temos o [RioJob](http://riojob.com.br/)

**Comment by gabrieldarezzo** [comment]
Bora fazer um Crawler que pega todas as vagas e criar o maior portal de vagas do mundo!!!!
[Não pera...](http://economia.estadao.com.br/noticias/negocios,justica-condena-grupo-catho-por-roubo-de-dados,78486e)

Tem esses aqui tmb:
http://stackoverflow.com/jobs

O trampos achei inovador no inicio, ainda mais pra quem estava muito acostumado a utilizar aquelas plataforma quadradonas hehe.

Aqui em Sp/Capital o que me dava melhor retorno era o:
http://www.apinfo.com/apinfo/

> melhor pro candidato conhecer todos os milhões de sites, verificá-los um a um e fazer essa diferenciação do peso de maneira pessoal.

Concordo :)

**Comment by felipefialho** [comment]
Te entendo, mas é natural, a demanda de vagas é muito grande e automaticamente surgem vários serviços visando atingir esse público, cada uma com suas peculiaridades.

Particulamente gosto de concorrência, tende a aumentar o nível dos serviços.

**Comment by leandroruel** [comment]
no começo gostava do trampos, mas agora acho que as vagas de lá é mais de agência e empresas de comunicação e etc. O pior serviço desse tipo é do infojobs... so vaga com descrição cagada, aliás, a maioria dos prints de vagas absurdamente exageradas sai de lá.

**Comment by rubenmarcus** [comment]
pra mim o pior continua sendo o vagas.com.br, as melhores vagas Stackoverflow Jobs, Mas o que eu ando vendo mesmo independente do site de recrutamento é uma lista cada vez maior de requisitos, para um valor cada vez menor de salário.

**Comment by munizart** [comment]
@rubenmarcus Isso quando divulgam o salário, a maior parte das vagas que vejo só divulgam o infame listão e nem deixam uma dica de quanto reconhecimento planejam dar ao futuro colaborador...
